src/app/layouts/app-layout/app-layout.component.html

```diff
@@ -1,7 +1,12 @@
-<app-navbar></app-navbar>
+@if(!isSignup() && !isLogin() && !isVerifyEmail() && !isActivate()) {
```

Comment:
Y si movemos esto a una señal computada? Para dejar lo más bobo posible el HTML

```
// En tu .ts
authLayout = computed(() => isSignup() || isLogind() || isVerifyEmail || isActivate())
```

```suggestion
@if (!authLayout()) {
```